### ISSUE TITLE
Income Limits clean up

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17851]
## Description 
Some guidance and patterns have been updated since [Income limits](https://www.va.gov/health-care/income-limits/introduction) has launched. These items should be updated to match the new VA guidelines:

1. Error message guidelines: "Please" should not be used in error messages anymore
   - "Please select a year." should be updated to "Select a year." 
   - "Please enter a valid 5 digit zip code." should be updated to "Enter a valid 5 digit zip code."
   - "Please enter a number between 0 and 100." should be updated to "Enter a number between 0 and 100."
2. Required indicator
   - On each screen, the question is the only thing for the user to do. For this reason we can remove the red, visual "Required" indicator from each question screen

## Acceptance Criteria
- [x] All error messages have been updated to remove the word "Please" 
- [x] "Required" has been removed visually from each question
- [ ] Requires design review
- [x] Requires accessibility review

**Comment by randimays** [comment]
@laflannery To confirm: this also applies to the service error messaging Income Limits uses? (Screenshot of the DST component with our text in it for ease of comparison):

Current:
<img width="1024" alt="Screenshot 2024-04-23 at 4 15 43 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/b50e52d1-48d4-4eb5-a07c-adddf2bb8bf6">

Expected:
<img width="1022" alt="Screenshot 2024-04-23 at 4 15 54 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/0b4937ac-de25-42ee-afd6-dc6567c4f17d">

**Comment by laflannery** [comment]
@randimays This guidance hasn't been formalized/added to the DS site yet but it is something that has come up during staging review and such but [I have asked the CAIA folks](https://dsva.slack.com/archives/C01K37HRUAH/p1713962973609779?thread_ts=1713544348.805399&cid=C01K37HRUAH) about this specifically. This specific message seems harsh without "Please" but that's probably just be me, so I want to be sure.

**Comment by laflannery** [comment]
@randimays I heard back from Laura Willwerth on CAIA and if possible she had some other additional suggestions for updating this particular error message I'm just not sure if we would be able to accomodate them:

> - We try to be more specific about the error when possible, so the header on a system error like this would usually say something like "We can't [xyz intended action, like 'access your records'] right now"
> - In new error messages we avoid subtle idioms like "run into a problem" and "on our end," so the body text might say something like "We're sorry. There's a problem with our system. Try again later."
> - We try to provide a recovery step beyond "try again" when possible—that's usually a phone contact number specific to the tool or form

I don't 100% know when this comes up, is it when we try to get to the review page or could it also happen after I try to submit my zip code or other fields? And to try again, can the user just try the continue button right there on the screen they are on or would they need to go back to the beginning of the form?

**Comment by laflannery** [comment]
@randimays I created [a new ticket for the system error](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17944)

**Comment by randimays** [comment]
Successfully validated the current and past flows in production. Closing!
